pyaedt/modeler/Circuit.py

```diff
@@ -112,17 +112,21 @@ def __init__(self, parent):
         self.layers = Layers(parent, self, roughnessunits="um")
         self._primitives = Primitives3DLayout(self._parent, self)
         self._primitivesDes = self._parent.project_name + self._parent.design_name
-        edb_folder = os.path.join(self._parent.project_path, self._parent.project_name + ".aedb")
-        edb_file = os.path.join(edb_folder, "edb.def")
-        if os.path.exists(edb_file):
-            self._mttime = os.path.getmtime(edb_file)
-            try:
-                self._edb = Edb(edb_folder, self._parent.design_name, True, self._parent._aedt_version, isaedtowned=True,
-                                oproject=self._parent.oproject)
-            except:
-                self._edb = None
-        else:
-            self._mttime = 0
+        # edb_folder = os.path.join(self._parent.project_path, self._parent.project_name + ".aedb")
+        # edb_file = os.path.join(edb_folder, "edb.def")
+        # if os.path.exists(edb_file):
+        #     self._mttime = os.path.getmtime(edb_file)
+        #     _main = sys.modules['__main__']
+        #     if "isoutsideDesktop" in dir(_main) and not _main.isoutsideDesktop and self._parent.oproject.GetEDBHandle():
+        #         try:
+        #             self._edb = Edb(edb_folder, self._parent.design_name, True, self._parent._aedt_version, isaedtowned=True,
+        #                             oproject=self._parent.oproject)
+        #         except:
+        #             self._edb = None
+        #     else:
+        #         self._edb = None
+        # else:
+        #     self._mttime = 0
```

Comment:
May consider just removing rather than commenting this.  If we do comment it, we might consider adding a note why we commented this and kept it around.

Reply:
I added a todo into def edb just below. I'll check if this code is still needed and eventually remove. right now it doesn't work in Ironpython

Reply:
If it adds functionality, we can have a branch that checks if in IronPython and then doesn't execute.